docs/core/compatibility/unsupported-apis.md

```diff
@@ -0,0 +1,373 @@
+
+> [!div class="mx-tdBreakAll"]
+> | Member | Platform |
+> | - | - |
+> | <xref:System.Security.Authentication.ExtendedProtection.ExtendedProtectionPolicy.%23ctor(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)?displayProperty=nameWithType> | All |
+
+## System.Security.Policy
+
+> [!div class="mx-tdBreakAll"]
+> | Member | Platform |
+> | - | - |
+> | <xref:System.Security.Policy.Hash.GetObjectData(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)?displayProperty=nameWithType> | All |
+
+## System.ServiceProcess.ServiceController
+
+> [!div class="mx-tdBreakAll"]
+> | Member | Platform |
+> | - | - |
+> | <xref:System.ServiceProcess.TimeoutException.%23ctor(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)?displayProperty=nameWithType> | All |
+
+## System.Text.RegularExpressions
+
+> [!div class="mx-tdBreakAll"]
+> | Member | Platform |
+> | - | - |
+> | <xref:System.Text.RegularExpressions.Regex.CompileToAssembly%2A?displayProperty=nameWithType> | All |
+
+## System.Threading
+
+> [!div class="mx-tdBreakAll"]
+> | Member | Platform |
+> | - | - |
+> | <xref:System.Threading.CompressedStack.GetObjectData(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)?displayProperty=nameWithType> | All |
+> | <xref:System.Threading.ExecutionContext.GetObjectData(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)?displayProperty=nameWithType> | All |
+> | <xref:System.Threading.Thread.Abort%2A?displayProperty=nameWithType> | All |
+> | <xref:System.Threading.Thread.ResetAbort?displayProperty=nameWithType> | All |
+> | <xref:System.Threading.Thread.Resume?displayProperty=nameWithType> | All |
+> | <xref:System.Threading.Thread.Suspend?displayProperty=nameWithType> | All |
+
+## System.Xml
+
+> [!div class="mx-tdBreakAll"]
+> | Member | Platform |
+> | - | - |
+> | <xref:System.Xml.XmlDictionaryReader.CreateMtomReader(System.Byte[],System.Int32,System.Int32,System.Text.Encoding[],System.String,System.Xml.XmlDictionaryReaderQuotas,System.Int32,System.Xml.OnXmlDictionaryReaderClose)?displayProperty=nameWithType> | All |
+> | <xref:System.Xml.XmlDictionaryReader.CreateMtomReader(System.IO.Stream,System.Text.Encoding[],System.String,System.Xml.XmlDictionaryReaderQuotas,System.Int32,System.Xml.OnXmlDictionaryReaderClose)?displayProperty=nameWithType> | All |
+> | <xref:System.Xml.XmlDictionaryWriter.CreateMtomWriter(System.IO.Stream,System.Text.Encoding,System.Int32,System.String,System.String,System.String,System.Boolean,System.Boolean)?displayProperty=nameWithType> | All |
+
+## System.Xml.Xsl
+
+> [!div class="mx-tdBreakAll"]
+> | Member | Platform |
+> | - | - |
+> | <xref:System.Xml.Xsl.XslCompiledTransform.Load%2A?displayProperty=nameWithType> (if `msxsl:script` is used in the stylesheet) | All |
```

Comment:
should you link to a specific overload or is this true for all overloads?

Reply:
@mairaw It's actually just 2 of 8 overloads, but after looking at the code, I'm not sure whether they should be listed here at all, because [they don't always throw the exception](https://github.com/dotnet/runtime/blob/5ef367a888fa8a25ce157cf340bd271efd36c72f/src/libraries/System.Private.Xml/src/System/Xml/Xslt/XslCompiledTransform.cs#L211). I initially got the info [here](https://gist.github.com/mairaw/c5ea693b956b584f44c3423e29e1f0c6#systemxmlxsl).

Reply:
Yeah they might belong in the breaking changes doc. This page is about always throwing PNSE right?

Reply:
That's right. And with the default compilation, as far as I can tell, the PNSE will actually never be thrown because `FEATURE_COMPILED_XSL` is [defined in the project file](https://github.com/dotnet/runtime/blob/25d316cefafc4c3884f38acf7bdef971d2b9fe4b/src/libraries/System.Private.Xml/src/System.Private.Xml.csproj#L7). But I'm not that familiar with preprocessor directives.

I'll remove this from this file and inquire about adding it to breaking changes.

Reply:
Not sure if you have this link but this is handy for finding owners: https://github.com/dotnet/corefx/blob/master/Documentation/project-docs/issue-guide.md#areas